### ISSUE TITLE
Adding rooms from a space?

### ISSUE THREAD

**Issue by CactiChameleon9** [SchildiChat/schildichat-android-next#36]
**Is your feature request related to a problem? Please describe.**
Would be nice to be able to see a list of rooms and subspaces that belong to a space (and the ability to join them from there)

**Describe the solution you'd like**
A page, maybe accessed through the hamburger menu, where you can see the space's information and rooms 

**Describe alternatives you've considered**
Low priority list of all available rooms on the main room lists for the respective spaces

**Comment by bugreporter005** [comment]
Any updates on this?

**Comment by Ferdi265** [comment]
This is quite an important feature to have; The [matrix.org client list](https://matrix.org/ecosystem/clients/schildichat-next/) lists SchildiChat Next as having spaces support, which can't really be said until this is implemented.

**Comment by DJCrashdummy** [comment]
interesting... i'm definitely on 0.10.10-EX_25_10_0 (fdroid build) and i still only see the rooms i've joined in lists accessed via Schildis space bar. 😦 

@Ferdi265 is there a setting for that or something else i have missed? 🤨

**Comment by SpiritCroc** [comment]
Yeah it is not intended as a proper solution. I made the EleX implementation accessible since it already exists, but it is in no way good UX mixing those two / relying on the EleX implementation to do certain tasks. But due to lack of time and other priorities it may be good enough for now.
